src/shared/components/VideoCategoryCard/VideoCategoryCard.style.ts

```diff
@@ -0,0 +1,147 @@
+import { css } from '@emotion/react'
+import styled from '@emotion/styled'
+import { transparentize } from 'polished'
+
+import { colors, sizes, transitions } from '@/shared/theme'
+
+import { Text } from '../Text'
+
+type ColorProps = { color: string }
+type LoadingProps = { loading?: boolean }
+type VariantProps = { variantCategory?: 'default' | 'compact' }
+
+export const CoverImg = styled.div<{ bgImgUrl: string }>`
+  position: relative;
+  width: 100%;
+  height: 100%;
+  background: url(${({ bgImgUrl }) => bgImgUrl});
+  background-position: center;
+  background-size: cover;
+`
```

Comment:
I'm wondering - is there any benefit to using div with `background` instead of `img` here?

Reply:
Tried an img first but it didn't fit well `background-position: center;` and `background-size: cover;` help

Reply:
I won't be able to tell much without any details